### ISSUE TITLE
NodeJS dependency

### ISSUE THREAD

**Issue by PhilGarb** [chrisjpatty/flume#52]
When trying to use the library with snowpack as a build tool I noticed that nanoid is required in a couple of places. 

```javascript
const nanoid = require("nanoid")
```

I tried to replace it in a couple of places with an esm import and it works.

```javascript
import { nanoid } from "nanoid"
```

This seems to work apart from nanoid depending on a nodejs package called crypto. Nanoid can however also be imported from: 
```javascript
import { nanoid } from "nanoid/non-secure/index/
```

From what I can see nanoid is used to generate a random ID which has no need to be secure (See [nanoid security](https://github.com/ai/nanoid#security)). By using the non-secure package and importing instead of requiring it the node dependency could be dropped making the library usable outside of a node bundler like webpack.

Is there something else I don't see? If not I could easily provide a PR to change this.

**Comment by chrisjpatty** [comment]
@PhilGarb Hey thanks for catching this! This looks like the right way to handle things, and you're right, it's just being used to generate IDs. I'd definitely welcome a PR to make this change, and I'll get it merged in asap.

**Comment by chrisjpatty** [comment]
Fixed by PR #53
